ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -586,6 +586,37 @@ tests:
     test:
     - ref: fips-check-optional-operator-images-scan
     workflow: cucushift-installer-rehearse-aws-ipi-proxy
+- as: aws-ipi-proxy-fips-f360-compliance
```

Comment:
Do we want to run all the test with `f360` for all versions ?
For disable jobs, it will be `f999`. 
For non-disabled jobs, align it with other tests in the specific version. E.g, in 4.12 with `f60`, 4.14 with `f28`.